src/components/VariantMutations.tsx

```diff
@@ -122,12 +125,13 @@ export const VariantMutations = ({ selector }: Props) => {
     const nucFetchQueue = new PromiseQueue();
     for (let nucElement of nuc) {
       nucFetchQueue.addTask(() =>
-        fetchUniquenessScore(nucElement, selector, variantCount, 'nuc').then(uniqueness =>
+        fetchUniquenessScore(nucElement, selector, variantCount, 'nuc').then(uniqueness => {
+          //console.log('flag 5')
```

Comment:
Please remove the commented-out code (there are a few more instances below).

Reply:
I did so